### DISCUSSION TITLE
Please publish ppa's or deb's for release builds.

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [OpenRCT2/OpenRCT2#16801] [Feature requests] [answered]
Please publish ppa's or deb's for release builds.
I want to beable to install openrct2 on my kde neon computer not just have it sitting in the downloads folder.

This is because ppa's and deb's...
1. it gets automatically added to the application menu (aka the start menu in windows)
2. it comes with icons
3. it doesn't clutter the downloads folder

**Comment by YJSoft** [comment, marked as answer]
https://launchpad.net/~openrct2/+archive/ubuntu/master - Release
https://launchpad.net/~openrct2/+archive/ubuntu/nightly - Nightly

**Comment by Shadowblitz16** [comment]
@YJSoft the master ppa just installs a debug build

**Reply by ocalhoun6** [reply]
I've been using the nightly PPA -- installed according to the instructions [here](https://openrct2.org/quickstart/install/linux/debian-ubuntu-mint) --  for a long time, and it works perfectly. Automatic updates and everything.

... Actually, scratch the 'works perfectly' part. There *is* still an issue of it not reporting the build/version numbers in-game ... but that's really a small issue.
